core/tests/performance_framework/perf_domain.py

```diff
@@ -35,19 +35,41 @@ class PageSessionMetrics(object):
     loading and includes the following keys:
 
         timing: maps to a dict containing the keys (all in milliseconds):
+            connectEnd: the Unix time the user agent finishes establishing the
+                connection to the server to retrive the current document.
+            connectStart: the Unix time before the user agent starts
+                establishing the connection to the server to retrive the
+                document.
+            domainLookupEnd: the Unix time the domain lookup ends.
+            domainLookupStart: the Unix time the domain lookup starts.
+            domComplete: the Unix time user agent sets the current document
+                readiness to "complete".
             domInteractive: the Unix time the the parser finished its work on
-                            the main document
-            domLoading: the Unix time the DOM started loading
-            fetchStart: the Unix time the page began loading
-            loadEventEnd: the Unix time the page finished loading
+                            the main document.
+            domLoading: the Unix time the DOM started loading.
+            fetchStart: the Unix time the page began loading.
+            loadEventEnd: the Unix time the load event of the current document
+                is completed.
+            loadEventStart: the Unix time the load event of the current document
+                is fired.
+            navigationStart: the Unix time  the prompt for unload terminates
+                on the previous document in the same browsing context
+            redirectEnd: the Unix time the last HTTP redirect is completed, that
+                is when the last byte of the HTTP response has been received
+            redirectStart: the Unix time the first HTTP redirect starts.
             requestStart: the Unix time the request started
```

Comment:
Nit: Should end with a period? Ditto for other entries. Please make sure all either end or don't end with periods (for consistency).